lib/api/controllers/security.js

```diff
@@ -305,7 +305,7 @@ class SecurityController extends NativeController {
       description,
       { apiKeyId, creatorId, refresh });
 
-    this.kuzzle.log.info(`[SECURITY] User "${request.context.user._id}" applied action "${request.input.action}" on user "${userId}."`);
+    this.kuzzle.log.info(`[SECURITY] User "${this.getUserId(request)}" applied action "${request.input.action}" on user "${userId}."`);
```

Comment:
you should use the creatorId variable instead (which should use that new getUserId function to prevent an error if invoked by a plugin)